### ISSUE TITLE
IPv6 support

### ISSUE THREAD

**Issue by tambry** [videojs/video.js#5263]
- [ ] IPv6 on website
- [x] IPv6 on vjs.zencdn.net
- [ ] IPv6 on a.zencdn.net
- [ ] HTTP/2 on a.zencdn.net

## Description
Both the website and resources from the CDN can't be loaded over IPv6.

### For website
1. Try to access the website with only IPv6 connectivity available. This can be simulated on an IPv6-capable connection using `curl "https://videojs.com/" -6` or by temporarily disabling IPv4.

### For CDN
1. Try to access any resource from the CDN using IPv6, e.g. `curl "https://vjs.zencdn.net/7.0.3/video.js" -6`

### Expected
Both the website and various resources from the CDN can be loaded using IPv6.

### Actual
Neither can be loaded over IPv6 at all.

### Error output
When accessing website with Chrome and only IPv6 connectivity available: `DNS_PROBE_FINISHED_NXDOMAIN`  
For forced IPv6 using curl:
* `curl: (6) Could not resolve host: videojs.com`
* `curl: (6) Could not resolve host: vjs.zencdn.net`

### CDN
Change to CNAME record on `vjs.zencdn.net` from `osff.map.fastly.net` to `dualstack.osff.map.fastly.net`.

The CDN used by VJS seems to be Fastly, which has good IPv6 support. The `vjs.zencdn.net` is currently CNAME'd to `osff.map.fastly.net`, which only has an IPv4 address. Strangely enough, IPv6 seems to be already enabled for the CDN map in question, and thus there exists `dualstack.osff.map.fastly.net`, which supports both IPv4 and IPv6.
The following works both for IPv4 and IPv6: `curl "https://dualstack.osff.map.fastly.net/7.0.3/video.js" -H
"Host:vjs.zencdn.net" -k`

### Website
~~The website seems to be hosted on the Google Cloud Platform, which at this time only has IPv6 support for load balancers. If the website uses a load balancer, it should be a simple configuration change. If it's a instance, then it'll be a matter of waiting, as [they're working on IPv6 support for them](https://googlecloudplatform.uservoice.com/forums/302595-compute-engine/suggestions/8518246-support-ipv6).~~

Netlify is working on IPv6 support and has told me that they'll have it by the end of the year.

**Comment by gkatsev** [comment]
Our website is hosted via netlify, which as far as I can tell, hasn't rolled out ipv6 support.

As for our CDN, it's hosted on fastly, I'll take a look at what I can do there.

**Comment by gkatsev** [comment]
Thanks for the reminder. I pinged fastly support to make sure that there isn't anything that we'd be overlooking in making the CNAME record switch.

**Comment by gkatsev** [comment]
I've made the change to fastly, it may take a while to propagate and what not.

**Comment by tambry** [comment]
Can confirm that vjs.zencdn.net now works over IPv6. Thanks!

I now noticed that there also seems to be a a.zencdn.net endpoint, which also isn't IPv6-enabled (it additionally lacks HTTP/2). Since the endpoint is using Akamai, I believe this should also be as easy as a switch in the control panel or pinging Akamai support. Maybe you could look into this?

**Comment by gkatsev** [comment]
cool, thanks for confirming. I guess I'm going to close this one since we resolved the CDN part and are just waiting on Netlify. Hopefully, we won't need to do anything on that side of things but if we do, please open another issue.

As for a.zencdn.net, that's unrelated to Video.js and shouldn't pose any issues. If that url is still being used, the team is probably already aware of IPV6 support issues.
